ttnn/cpp/ttnn/deprecated/tt_dnn/op_library/moreh_getitem/moreh_getitem_op.cpp

```diff
@@ -26,22 +26,22 @@ void MorehGetitem::validate_with_output_tensors(
     TT_FATAL(input_tensor.storage_type() == StorageType::DEVICE, "Operands to getitem need to be on device!");
     TT_FATAL(input_tensor.buffer() != nullptr, "Operands to getitem need to be allocated in buffers on device!");
     auto dtype = input_tensor.get_dtype();
-    TT_FATAL(dtype == DataType::INT32 || dtype == DataType::BFLOAT16);
+    TT_FATAL(dtype == DataType::INT32 || dtype == DataType::BFLOAT16, "Error");
```

Comment:
not a useful error message

Reply:
I automated and used sed. The condition will be printed.